main.js

```diff
@@ -32,3 +32,20 @@ function start() {
   }
 }
 start();
+
+var divElement = document.querySelector(".week");
+divElement.addEventListener("click", daySchedule);
+
+function daySchedule() {
+  if(event.target === null) {
+    return;
+  }
+  else if(event.target.className === "day") {
+    var text = event.target.textContent;
+    var emElement = document.querySelector("em");
+    emElement.textContent = text;
+  }
+  else {
+    return;
+  }
+}
```

Comment:
It won't ever be null, You can just stick to this
```
if(event.target.className === "day") {
    var text = event.target.textContent;
    var emElement = document.querySelector("em");
    emElement.textContent = text;
  }
```
Or even better
```
if(event.target.className !== 'day') return;
    var text = event.target.textContent;
    var emElement = document.querySelector("em");
    emElement.textContent = text;
```
You should make emElement a span element with a class thats more easier to understand, as  your code reader i have no idea what emElement means

It should also be a global variable so you don't have to query the dom every time.